site/planet.ini

```diff
@@ -0,0 +1,11 @@
+title = Byld
+
+[Viresh_GSoC]
+  title = Viresh Gupta, https://avatars0.githubusercontent.com/u/10978108?s=460&v=4
+  link  = https://virresh.wordpress.com/tag/gsoc/
```

Comment:
We aren't using the link to blog anywhere it seems. Should clicking on a person's photo or a name take us to their blog? 

This could be a separate issue

Reply:
Please see the PR description and this too : https://github.com/IIIT-Delhi/byld-website/pull/62/files/4e7c160242dcd6d8ef64120d226d6c832392c3e5#r221629532